app/models/emoji.rb

```diff
@@ -60,6 +60,7 @@ def self.clear_cache
     Discourse.cache.delete(cache_key("standard_emojis"))
     Discourse.cache.delete(cache_key("aliases_emojis"))
     Discourse.cache.delete(cache_key("all_emojis"))
+    Discourse.cache.delete(cache_key("tonable_emojis"))
```

Comment:
It certainly would be enough, it was just an opportunity to avoid repeating the "string" at multiple places, but your solution totally works for me.